features-support/step_definitions/features.js

```diff
@@ -85,13 +92,13 @@ module.exports = function () {
         return world.browser.findElement(By.id(repositoryCongtrolsId));
 
         // Get the repo controls element.
-      }, handleErr(callback))
```

Comment:
I have the vague memory that the explicit error handler was required because of the way WebDriver implemented support for promises, have you checked this setup correctly handles the error when e.g. WebDriver fails to find an element? They may well have fixed the implementation or I might just have got in wrong in the first place. I'm sorry to butt in on your excellent work 😉